.gitignore

```diff
@@ -39,4 +39,6 @@ buildNumber.properties
 
 *.idea
 *.iml
-*.MF
\ No newline at end of file
+*.MF
+Trash.java
```

Comment:
instead of adding this to gitignore, don't add files into repository. you may also remove them: https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-a-git-repository